storage/innobase/xtrabackup/src/xbstream.c

```diff
@@ -187,8 +191,8 @@ static
 void
 print_version(void)
 {
-	printf("%s  Ver %s for %s (%s)\n", my_progname, XBSTREAM_VERSION,
-	       SYSTEM_TYPE, MACHINE_TYPE);
+	printf("%s  Ver %s for %s (%s) (revision id: %s)\n", my_progname, XBSTREAM_VERSION,
```

Comment:
Do we print revision id for xtrabackup?

Reply:
we do.. saw your output above

Reply:
Yeah, the idea is that we have the same version schema across all tools. The only difference is that those auxiliary tools are not based on MySQL, so we don't report that. But the version and commit ID should match. This will help in troubleshooting to ensure the user has the correct version.